app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -5,7 +5,7 @@
     android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?attr/mainBackground"
+    android:background="@color/black"
```

Comment:
Why have we changed this, is this done, supporting the themes?

Reply:
I also don't understand why this change is made.